### PR TITLE
Add color balance(Hue, Saturation, Brightness, Contrast) adjustment

### PR DESCRIPTION
**Pull request by feiwan1**
on VA module.

Jira: None.
Test: Build passes on Linux and Android.
Signed-off-by: Wang,Fei <fei.w.wang@intel.com>

### CODE REVIEW

File: common/compositor/va/varenderer.cpp [hunk 1]
```diff
@@ -294,6 +348,49 @@ bool VARenderer::Draw(const MediaState& state, NativeSurface* surface) {
   param.filters = nullptr;
   param.filter_flags = VA_FRAME_PICTURE;
 
+  VAProcFilterCapColorBalance vacaps[VAProcColorBalanceCount];
+  uint32_t vacaps_num = VAProcColorBalanceCount;
+
+  if (caps_.empty()) {
+    if (!QueryVAProcFilterCaps(
+           va_context, VAProcFilterColorBalance, vacaps, &vacaps_num))
```
Comment (kalyankondapally) [inline common/compositor/va/varenderer.cpp]: brackets for if and else

**Comment by kalyankondapally** [comment]
@XiaosongWei You want to review this.

**Comment by feiwan1** [comment]
@kalyankondapally @XiaosongWei , re-pushed this patch, please have a review.

**Comment by XiaosongWei** [comment]
lgtm

**Comment by kalyankondapally** [comment]
thanks merged

**Pull request closed without merging**